src/devices/tuya.ts

```diff
@@ -2335,15 +2335,39 @@ export const definitions: DefinitionWithExtend[] = [
         model: "CK-BL702-AL-01",
         vendor: "Tuya",
         description: "Zigbee LED bulb",
+
+        // Add standard Zigbee power on behavior converters
+        toZigbee: [tz.power_on_behavior],
+        fromZigbee: [fz.power_on_behavior],
+
         extend: [
             tuya.modernExtend.tuyaLight({
                 colorTemp: {range: [142, 500]},
                 color: true,
             }),
         ],
+
```

Comment:
```suggestion
exposes: [e.power_on_behavior(["off", "on", "toggle", "previous"])],
```

This should be enough, it's automatically merged with the exposes from `tuya.modernExtend.tuyaLight`